src/App.tsx

```diff
@@ -1,39 +1,63 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './Components/Clock/Clock';
+
+interface State {
+  hasClock: boolean,
+  clockName: number,
+}
 
 function getRandomName(): string {
```

Comment:
In this function, you do not return a whole name, but only a random value
```suggestion
function getRandomValue(): string {
```